index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navItems">
+     
+        
+        <div class="images">
+         <div class="space">
+            <div class="image01"><img src="assets/Shape.png" class="shape00"></div>
+            <div class="image02"><img src="assets/Shape (1).png" class="shape01"></div>
+            </div>
+            <div class="menuitem">
+             <div class="boxic"><img src="assets/ic_menu.svg" class="ic_menu"></div>  
+               </div>
+
+        <div></div>
+
+        
+  </header>
+
+  <section class="container">
+       
+    <div class=" box name">Samuel Ackon</div>
+    <div class=" box intro">Hello! I am a software developer! 
+      I can help you build a product, feature or website. 
+      Taka a look of my works. 
+      If you like what you see and have a 
+      project you need coded,
+       don’t hesitate and contact me.</div>
+       <div class="rectangle">
+          <div>Start collaboration</div></div>
+
+       <div class="designs">
+          <div class="image3"><img src="assets/image_geometry_2.png" class="shape002"></div>
+      </div>
+ 
+
+
+
+</section>
```

Comment:
Please note that you must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Your design:
![your-design](https://user-images.githubusercontent.com/10332499/148390891-08a23910-4146-44aa-b712-c5fcc8d916cd.PNG)

### Figma design:
![figma](https://user-images.githubusercontent.com/10332499/148390938-68771ffd-d07e-4310-95ab-0bd915fa846b.PNG)